apps/admin-ui/src/component/reservations/EditTimeModal.tsx

```diff
@@ -280,44 +268,43 @@ const DialogContent = ({ reservation, onAccept, onClose }: Props) => {
   const translateError = (errorMsg?: string) =>
     errorMsg ? t(`reservationForm:errors.${errorMsg}`) : "";
 
-  const newTime = `${reservationDateTime(
-    newStartTime,
-    newEndTime,
-    t
-  )}, ${reservationDuration(newStartTime, newEndTime)} t`;
+  const newTimeString =
+    start && end
+      ? `${reservationDateTime(
+          start,
+          end,
+          t
+        )}, ${reservationDuration(start, end)} t`
+      : "";
 
   const originalTime = `${reservationDateTime(
     startDateTime,
     endDateTime,
     t
   )}, ${reservationDuration(startDateTime, endDateTime)} t`;
 
-  const recurringReservationInfo = reservation.recurringReservation ? (
-    <TimeInfoBox>
-      {t("Reservation.EditTime.recurringInfoLabel")}:{" "}
-      <Bold>
-        {recurringReservationInfoText({
-          weekdays: filterNonNullable(
-            reservation.recurringReservation.weekdays
-          ),
-          // begin: reservation.recurringReservation.beginDate ?? undefined,
-          begin: ((x) => (x != null ? new Date(x) : undefined))(
-            reservation.recurringReservation.beginDate
-          ),
-          // end: reservation.recurringReservation.endDate ?? undefined,
-          end: ((x) => (x != null ? new Date(x) : undefined))(
-            reservation.recurringReservation.endDate
-          ),
-          t,
-        })}
-      </Bold>
-    </TimeInfoBox>
-  ) : null;
-
   return (
     <Dialog.Content>
       <StyledForm onSubmit={handleSubmit(onSubmit)} noValidate>
-        {recurringReservationInfo}
+        {reservation.recurringReservation ? (
+          <TimeInfoBox>
+            {t("Reservation.EditTime.recurringInfoLabel")}:{" "}
+            <Bold>
+              {recurringReservationInfoText({
+                weekdays: filterNonNullable(
+                  reservation.recurringReservation.weekdays
+                ),
+                begin: ((x) => (x != null ? new Date(x) : undefined))(
+                  reservation.recurringReservation.beginDate
+                ),
+                end: ((x) => (x != null ? new Date(x) : undefined))(
+                  reservation.recurringReservation.endDate
+                ),
+                t,
+              })}
+            </Bold>
+          </TimeInfoBox>
+        ) : null}
```

Comment:
Couldn't this be simply `reservation.recurringReservation && <TimeInfoBox... />`? Or is there a reason to use a ternary expression with `null`?